### PR TITLE
fix: api date picker to drop down

### LINKED ISSUE #9760: API key expiration time calender is out of theme, text is not visible dark mode
### Issue Summary

Dates are not visible in dark mode in calender in cal.com API section in dark mode

### Steps to Reproduce

1. Click on settings on bottom left
2. Select API keys from side bar
3. Try to choose expiration time from calender

or if you're logged in you can simpley click this [link](https://app.cal.com/settings/developer/api-keys)
Screenshot 2023-06-23 at 10 16 31 PM
pp.cal.com/settings/developer/api-keys)

### PR DESCRIPTION
**Pull request by AnitKrJha**
## What does this PR do?

Changes the date-picker to a 4 option dropdown for setting expiration date of api_key

Fixes #9760

## Before
image

## After

https://github.com/calcom/cal.com/assets/103268398/a973931b-a114-44d7-b8cd-039fe0411fb0

  
/claim #9760

### CODE REVIEW

**Review by AnitKrJha — COMMENTED** [review]
Feel good to go with

**Comment by AnitKrJha** [comment]
@PeerRich please review the changes asap!

File: packages/features/ee/api-keys/components/ApiKeyDialogForm.tsx [hunk 1]
```diff
@@ -150,15 +181,37 @@ export default function ApiKeyDialogForm({
...
+                      }
+                      onChange(option.value);
+                      setExpiryDate(option.value)
+                    }}
+                    defaultValue={defaultValue}
                   />
-                )}
+                )}}
               />
+              <span className="text-subtle text-xs mt-2">The API key will expire on <span className="font-bold"> {dayjs(expiryDate).format('DD-MM-YYYY')}</span></span>
```
Comment (devj3ns) [inline packages/features/ee/api-keys/components/ApiKeyDialogForm.tsx:214]: I guess this text should be translated, right?

Comment (AnitKrJha) [reply packages/features/ee/api-keys/components/ApiKeyDialogForm.tsx]: @devj3ns I fixed it can you review it!! ASAP

**Comment by PeerRich** [comment]
looks good so far! @CarinaWolli can give a final say

**Comment by PeerRich** [comment]
or actually maybe @sean-brydon

File: packages/features/ee/api-keys/components/ApiKeyDialogForm.tsx [hunk 2]
```diff
@@ -150,15 +181,37 @@ export default function ApiKeyDialogForm({
...
                 name="expiresAt"
-                render={({ field: { onChange, value } }) => (
-                  <DatePicker
-                    disabled={watchNeverExpires || !!defaultValues}
-                    minDate={new Date()}
-                    date={value}
-                    onDatesChange={onChange}
+                render={({ field: { onChange, value } }) => {
+                  const defaultValue = expiresAtOptions[1]
+                  console.log(defaultValue,value,expiresAtOptions);
```
Comment (sean-brydon) [inline packages/features/ee/api-keys/components/ApiKeyDialogForm.tsx:186]:
```suggestion

```

**Review by sean-brydon — DISMISSED** [review]
Can't fault seems to work well!

**Review by sean-brydon — APPROVED** [review]

**Pull request merged**